doc/userguide/rules/flow-keywords.rst

```diff
@@ -318,90 +318,61 @@ Signature example::
 
 In this example, we combine `flow.age` and `flowbits` to get an alert on the first packet after the flow's age is older than one hour.
 
-flow.pkts_toclient
-------------------
+flow.pkts
+---------
 
-Flow number of packets to client (integer)
+Flow number of packets (integer)
 This keyword does not wait for the end of the flow, but will be checked at each packet.
 
-flow.pkts_toclient uses an :ref:`unsigned 32-bit integer <rules-integer-keywords>`.
+flow.pkts uses an :ref:`unsigned 32-bit integer <rules-integer-keywords>` and supports
+following directions:
 
-Syntax::
-
- flow.pkts_toclient: [op]<number>
-
-The number of packets can be matched exactly, or compared using the _op_ setting::
-
- flow.pkts_toclient:3    # exactly 3
- flow.pkts_toclient:<3   # smaller than 3
- flow.pkts_toclient:>=2  # greater than or equal to 2
+* toclient
 
-Signature example::
+* toserver
 
- alert ip any any -> any any (msg:"Flow has 20 packets"; flow.pkts_toclient:20; sid:1;)
-
-flow.pkts_toserver
-------------------
-
-Flow number of packets to server (integer)
-This keyword does not wait for the end of the flow, but will be checked at each packet.
-
-flow.pkts_toserver uses an :ref:`unsigned 32-bit integer <rules-integer-keywords>`.
+* either
 
 Syntax::
 
- flow.pkts_toserver: [op]<number>
+ flow.pkts:<direction>,[op]<number>
 
 The number of packets can be matched exactly, or compared using the _op_ setting::
 
- flow.pkts_toserver:3    # exactly 3
- flow.pkts_toserver:<3   # smaller than 3
- flow.pkts_toserver:>=2  # greater than or equal to 2
+ flow.pkts:toclient,3    # exactly 3
+ flow.pkts:toserver,<3   # smaller than 3
+ flow.pkts:either,>=2  # greater than or equal to 2
 
 Signature example::
 
- alert ip any any -> any any (msg:"Flow has 20 packets"; flow.pkts_toserver:20; sid:1;)
-
-flow.bytes_toclient
```

Comment:
I guess doc should not be removed if the keyword still exist

You can just have a quick mention of it saying it has the same behavior as `flow.bytes: toclient,`

Reply:
ok. My idea of removing the doc for old syntax was to increase adaptation of the new syntax only. Old syntax is for backwards compatibility so the existing rulesets using it do not break and that's it. Lmk wdyt?

Reply:
@jufajardini thoughts ?

I see `base64_decode` doc has a sentence : `We recommend using the base64 transform instead`